string controllername
float64 dronemass
float64 cablelength
float64 a_j
float64 payloadmass
float64 motor_slope
float64 motor_intercept 
int32 num_drone
bool isPubAuxiliaryState
bool isAddonForcedUsed
bool isCrossFeedingTermsUsed
float64 t_jx
float64 t_jy
float64 t_jz
float64 kv_xy
float64 kv_z
float64 kR_xy
float64 kR_z
float64 kL
float64 Kphi_xy
float64 Kphi_z
float64 kr1_x
float64 kr1_y
float64 kr1_z
float64 kr2_x
float64 kr2_y
float64 kr2_z
float64 kp_x
float64 kp_y
float64 kp_z
float64 komega_x
float64 komega_y
float64 komega_z
float64 lambdaj_x
float64 lambdaj_y
float64 lambdaj_z
float64 lambda_T_x
float64 lambda_T_y
float64 lambda_T_z
float64 lambda_R_x
float64 lambda_R_y
float64 lambda_R_z
float64 lambda1_x
float64 lambda1_y
float64 lambda1_z
float64 lambda2_x
float64 lambda2_y
float64 lambda2_z
float64 pxy_error_max
float64 pz_error_max
float64 pxy_int_max
float64 pz_int_max
float64 tilt_max
float64 int_start_error
float64 fp_max_x
float64 fp_max_y
float64 fp_max_z
---
bool oktostart